packages/components/src/components/common/state-message/state-message.tsx

```diff
@@ -14,18 +14,19 @@ export const messageId = 'message';
 export const StateMessage: FunctionalComponent<StateMessageProps> = ({ state, message, theme, host }) => {
   const PrefixedTagNames = getPrefixedTagNames(host);
   const isErrorState = state === 'error';
+  const isSuccessState = state === 'success';
```

Comment:
to be in sync with other conditions in state-message, we could simply use `isErrorState` instead of `isSuccessState`